pkg/executables/config/clusterctl.yaml

```diff
@@ -81,6 +85,9 @@ images:
   infrastructure-docker/capd-manager:
     repository: {{.DockerManagerRepository}}
     tag: {{.DockerManagerTag}}
+  infrastructure-nutanix/manager:
+    repository: {{ .ClusterApiNutanixControllerRepository }}
+    tag: {{ .ClusterApiNutanixControllerTag }}
```

Comment:
nit: can you move this below `infrastructure-docker/kube-rbac-proxy` so the two docker ones are together

Also, does CAPX need `kube-rbac-proxy`? If so, you might wanna add that here as well

Reply:
Sure; We haven't added the KubeRBACProxy to the nutanix bundle yet. If needed, we can take care of that in a future PR.